### DISCUSSION TITLE
❌ Bug - Elementor contact form (Dynamic Tag)

### DISCUSSION THREAD

**Discussion by guy01992** [elementor/elementor#25619] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

When using the elementor contact form and using the dynamic tag for the TO email it does not work and defaults back to the admin email when filling out the contact form. The admin receives the form instead of the author.
I want to use the dynamic tag for author email. As the contact form is on the author page. 

### Describe the solution you'd like.

Needs fixing 

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by nicholaszein** [comment]
Hello @guy01992!

Please, remember to follow the [Contribution Guidelines](https://elemn.to/gh-contributing) for submitting Bug Reports. This space is reserved for Feature Requests only. Please use the following link to submit Bug Reports: https://elemn.to/gh-new-bug-report

Kind regards.
